perf/perf_sdt_probe.py

```diff
@@ -72,6 +72,7 @@ def add_library(self):
         self.run_cmd(perf_add)
         if self.is_fail:
             self.fail("Unable to add %s to builid-cache" % self.libpthread)
+        "sleep 10"
```

Comment:
ah, thing wrong, why sleep is required? and please have sleep using python way

Reply:
> 

@PraveenPenguin addressed review comment

sleep is required as we are comparing perf list before and after adding sdt probe event, in some cases test is failing as values are same and not reflecting.

Reply:
@disgoel sure , can you please wrap all commits in single commit